lib/Zyberspace/Telegram/Cli/Client.php

```diff
@@ -372,4 +372,34 @@ public function getHistory($peer, $limit = null, $offset = null)
 
         return $this->exec('history ' . $this->escapePeer($peer) . $limit . $offset);
     }
+
+    /**
+     * Send Picture to peer
+     * 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Whitespace found at end of line](https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/1970418740/issues/source?bid=2302274&fileBranchId=3113578#l378)